docs/src/pages/components/emptyState/code.mdx

```diff
@@ -0,0 +1,6 @@
+#### Live Demo
+<br/>
+
+<Preview name="ccomponents-emptystate-all--all" />
```

Comment:
Please check the story url. I think there is some typo in `ccomponents`

Reply:
done